src/pages/workspaces/workspace/analysis/file-utils.ts

```diff
@@ -65,17 +66,16 @@ export const useAnalysisFiles = (): AnalysisFileStore => {
   const [analyses, setAnalyses] = useState<AnalysisFile[]>([])
   const [pendingCreate, setPendingCreate] = useLoadedData<true>()
 
-  const refresh: () => Promise<void> = _.flow(
-    // @ts-expect-error
+  const refresh = withHandlers([
```

Comment:
the cast is no longer needed, and the type is reliably inferred now.... AWESOME!